test/integration/metrics/metrics_test.go

```diff
@@ -105,10 +105,7 @@ func TestAPIServerStorageMetrics(t *testing.T) {
 }
 
 func TestAPIServerMetrics(t *testing.T) {
-	// KUBE_APISERVER_SERVE_REMOVED_APIS_FOR_ONE_RELEASE allows for APIs pending removal to not block tests
-	t.Setenv("KUBE_APISERVER_SERVE_REMOVED_APIS_FOR_ONE_RELEASE", "true")
```

Comment:
leave this bit as-is, it doesn't hurt anything and we can clean it up when we stop using a real beta type for this test